.htaccess

```diff
@@ -42,7 +42,7 @@ deny from env=stayout
     RewriteRule src/Backend/Core/Js/ckfinder/core/connector/php/connector\.php - [L]
 
     # allow the php-opcache-reset.php file
-    RewriteRule ^php-opcache-reset\.php - [L]
+    RewriteRule ^php-opcache_reset\.php - [L]
```

Comment:
Why? Wouldn't it make a lot more sense to fix the filename? With this change it is a mix of - en _